Index: src/main/java/TomteNisse.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class TomteNisse {\r\n    String namn;\r\n    int years;\r\n\r\n    public TomteNisse(String namn, int years) {\r\n        this.namn = namn;\r\n        this.years = years;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/TomteNisse.java b/src/main/java/TomteNisse.java
--- a/src/main/java/TomteNisse.java	(revision dcc462939c97f28b798e3b1d4f2a44935cde205b)
+++ b/src/main/java/TomteNisse.java	(date 1767086556000)
@@ -1,9 +1,18 @@
 public class TomteNisse {
-    String namn;
+
+    String name;
     int years;
 
-    public TomteNisse(String namn, int years) {
-        this.namn = namn;
+    public TomteNisse(String name, int years) {
+        this.name = name;
         this.years = years;
     }
+
+    public String getName () {
+        return name;
+    }
+
+    public int getAge () {
+        return years;
+    }
 }
Index: src/main/java/GodJul.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class GodJul {\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/GodJul.java b/src/main/java/GodJul.java
--- a/src/main/java/GodJul.java	(revision dcc462939c97f28b798e3b1d4f2a44935cde205b)
+++ b/src/main/java/GodJul.java	(date 1767085170000)
@@ -1,3 +1,16 @@
 public class GodJul {
 
+    public static void main(String[] args) {
+
+
+        TomteNisse nisse1 = new TomteNisse("Nisse", 574);
+        TomteNisse nisse2 = new TomteNisse("Tomas", 420);
+        TomteNisse nisse3 = new TomteNisse("Tamara", 175);
+
+        System.out.println("Dessa nissar har tomten:");
+        System.out.println(nisse1.getName() + ", " + nisse1.getAge() + " år");
+        System.out.println(nisse2.getName() + ", " + nisse2.getAge() + " år");
+        System.out.println(nisse3.getName() + ", " + nisse3.getAge() + " år");
+
+    }
 }
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision dcc462939c97f28b798e3b1d4f2a44935cde205b)
+++ /dev/null	(revision dcc462939c97f28b798e3b1d4f2a44935cde205b)
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
